pyaedt/modeler/Object3d.py

```diff
@@ -642,30 +664,37 @@ def vertices(self):
 
     @property
     def m_Editor(self):
-        """Provides a pointer to the oEditor object in the AEDT API. Intended primarily for use by
-           FacePrimitive, EdgePrimitive, VertexPrimitive child objects
+        """Pointer to the oEditor object in the AEDT API. This property is 
```

Comment:
Don't worry too much about this property here.  It's likely going to be made private at some point.